sprints/dashboard/tasks.py

```diff
@@ -28,11 +30,14 @@
 )
 from sprints.dashboard.libs.jira import connect_to_jira
 from sprints.dashboard.models import Dashboard
+from sprints.webhooks.models import Webhook, WebhookEvent
```

Comment:
This should be below `sprints.dashboard.utils` (it's sorted alphabetically). Also `WebhookEvent` seems to be unused.